doc/connectivity/bluetooth/api/mesh/rpr_srv.rst

```diff
@@ -22,6 +22,13 @@ Note that after refreshing the device key, node address or Composition Data thro
 Interface (NPPI) procedure, the :c:member:`bt_mesh_prov.reprovisioned` callback is triggered. See section
 :ref:`bluetooth_mesh_models_rpr_cli` for further details.
 
+Limitations
+-----------
+
+* Remote Provisioning Server does not support provisioning of unprovisioned device using PB-GATT.
+* Remote Provisioning Server supports all Node Provisioning Protocol Interface procedures, however, if the composition of the device is changed as a result of device firmwae upgrade (See :ref:`firmware effect <bluetooth_mesh_dfu_firmware_effect>`) procedure, it is not possible for a device to remain provisioned. The device should be unprovisioned if composition data is expected to change.
+
```

Comment:
```suggestion
The following limitations apply to Remote Provisioning Server model:

* Provisioning of unprovisioned device using PB-GATT is not supported.
* All Node Provisioning Protocol Interface (NPPI) procedures are supported. However, if the composition data of a device has changed after a performed device firmware update (see :ref:`firmware effect <bluetooth_mesh_dfu_firmware_effect>`), it is not possible for the device to remain provisioned. The device should be unprovisioned if its composition data is expected to change.
```

Reply:
Proposal for second bullet point is making it unclear. Rephrasing it a bit for clarity.